swift/apple/FirezoneNetworkExtension/ConfigurationManager.swift

```diff
@@ -80,7 +80,17 @@ class ConfigurationManager {
   }
 
   @objc private func handleUserDefaultsChanged(_ notification: Notification) {
-    self.managedDict = userDefaults.dictionary(forKey: managedDictKey) ?? [:]
+    let newManagedDict = (userDefaults.dictionary(forKey: managedDictKey) ?? [:]) as [String: Any?]
+
+    // NSDictionary conforms to Equatable
+    if (managedDict as NSDictionary) != (newManagedDict as NSDictionary) {
+      Log.log("""
+        Applying MDM configuration:
```

Comment:
Not a fan of multi-line logs myself, IMO each log entry should be 1 line in a text file.

Reply:
These are still JSON formatted (for the file logger) and then os_log as well. I believe they format nicely in the Console.app since long lines will be truncated.

I can remove them though not a biggie.